package.json

```diff
@@ -7,6 +7,7 @@
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
+    "dependencies": "0.0.1",
```

Comment:
What's this for?

Reply:
Forgive me for this, earlier i used to type "npm install dependencies" instead of "npm install" after cloning a repo. Learned this thing yesterday

Reply:
Haha, no worries, it happens ✨

Also, you don't need to apologize for these small mistakes, you're contributing to open source in your free time, so I should be the one thanking you,

Just delete this line, and it should be alright